docs/components/Header.js

```diff
@@ -1,11 +1,13 @@
 // @flow strict
 import { type Node, useCallback, useEffect, useState } from 'react';
-import { Box, Flex, FixedZIndex, Text, Icon, IconButton, Sticky } from 'gestalt';
+import { Box, Flex, FixedZIndex, Text, IconButton, Sticky } from 'gestalt';
+import Image from 'next/image';
 import DocSearch from './DocSearch.js';
 import HeaderMenu from './HeaderMenu.js';
 import Link from './Link.js';
 import trackButtonClick from './buttons/trackButtonClick.js';
 import { useNavigationContext } from './navigationContext.js';
+import gestaltLogo from '../public/gestalt-logo-250.png';
```

Comment:
@dangerismycat could we use an inline SVG instead (Just like we do with our icons)? It would result in:

* Avoiding blurriness when zooming in:
![image](https://user-images.githubusercontent.com/127199/146345561-a6f93fdc-e0c6-4a2e-895f-0d95a22898b5.png)

* No longer flash the logo on load:
![logo-flash-on-load](https://user-images.githubusercontent.com/127199/146346032-7647dcdc-818a-4fce-ad76-a9919dd1ac72.gif)

Reply:
yep, will take care of that in a follow-up PR